appintro/src/main/res/values/styles.xml

```diff
@@ -3,22 +3,42 @@
 
     <style name="AppIntroStyle" parent="Theme.AppCompat.DayNight.NoActionBar" />
```

Comment:
I think we should rename this to be `AppIntroTheme` as it's actually a theme and not a style.
Moreover we can rename the Themes in the `example` module to avoid naming clashes

Reply:
Okay